assets/training/finetune_acft_image/environments/acft_image_mmdetection/context/requirements.txt

```diff
@@ -6,12 +6,11 @@ azureml-acft-image-components=={{latest-pypi-version}}
 azureml-core==1.54.0
 azure-ai-ml==1.11.0
 requests
-datasets==2.14.7
-transformers==4.34.1
-accelerate==0.23.0
-# NOTE: optimum is pinned to old version because with new version, we see big drop in run time performance for MMD
-optimum==1.8.8
-diffusers==0.20.2
+datasets==2.15.0
+transformers==4.36.2
+accelerate==0.25.0
+optimum==1.16.1
```

Comment:
The performance does not drop with this version anymore?

Reply:
Yes, it doesn't. Compared here.
New run: https://ml.azure.com/experiments/id/462fd9a6-fd52-4d2b-a296-dc29f96c4193/runs/2cd012a6-2c97-4a2f-a90d-1cee66efdee5?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourceGroups/nilesh-new-rg/providers/Microsoft.MachineLearningServices/workspaces/nilesh-ws-new&tid=72f988bf-86f1-41af-91ab-2d7cd011db47
FT time: 2h 25m
map: 0.2691

Old run: 
https://ml.azure.com/experiments/id/15c6d69f-f710-41db-95f9-bb912f1102a1/runs/3b4d89c3-1604-4d01-8d6f-4dfca01d7d9c?wsid=/subscriptions/381b38e9-9840-4719-a5a0-61d9585e1e91/resourcegroups/automlimage_eastus2_rg/workspaces/madhumaddi-aml-demo&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#
Ft time: 2h 32m
map: 0.286